.changeset/small-swans-approve.md

```diff
@@ -0,0 +1,5 @@
+---
+'@compiled/babel-plugin': minor
```

Comment:
@MonicaOlejniczak @at-nathan @JakeLane Hey gang, one thing we need to be careful about (at least when we go for wider adoption) is currently since the majority of Compiled's packages are pre 1.0 - each minor version as far as semver is concerned around de-duping, is a major version.

So if we don't have any API change - most changes should flow through patch changes.